docs/use-cases/schedule-a-workflow-run.md

```diff
@@ -0,0 +1,128 @@
+# Schedule a workflow run
+
+This guide shows you how to use Semaphore to set up the scheduled workflow runs
+in your project.
```

Comment:
Remove this line. Self-describing sentences give little or no value to the reader.